### ISSUE TITLE
Heat capacitor NA output for t>481min

### ISSUE THREAD

**Issue by SM6XSG** [ottopaulsen/node-red-contrib-power-saver#194]
If I put a time bigger than 481min into either Time +1C or Time -1C
![image](https://github.com/ottopaulsen/node-red-contrib-power-saver/assets/99121797/7a8491f2-42b0-4256-8d79-371319d5e36f)

 I get a NaN output to my temperature setpoint. Image below made with 600min setting for +1C as this room is super slow to heat up... 
![image](https://github.com/ottopaulsen/node-red-contrib-power-saver/assets/99121797/7fe4d818-6d6b-40f0-8788-ea836c8dc10b)

**Comment by TomTorger** [comment]
Yes, here you will run into an issue. 2 deg adjustment and 8 hours of Time + 1 deg, gives a 16 hour heating time. Our future prize horizon is quite often shorter than this and, hence, we will not be able to identify any "buy" window.

In similar cases, I have changed how to approach this: Try to find a period that matches the typical price variations in your area. Use the typical low price periods as the T1P and peak price periods as T1M. Then set the max T to 1 deg. This will allow you to exploit low price periods and avoid heating during peak prices.
